README.md

```diff
@@ -124,6 +124,13 @@ pip install pre-commit
 pre-commit install --allow-missing-config
 ```
 
-## Changelog
+## Showcase
 
-See [Changelog.md](https://github.com/sbrunner/jsonschema2md/blob/master/CHANGELOG.md).
+- [PrairieLearn's `infoCourse.json`](https://prairielearn.readthedocs.io/en/latest/schemas/infoCourse/). [Source code](https://github.com/PrairieLearn/PrairieLearn/blob/ab1e0f1fc837a8da9cde3448eb785958ac42e309/docs/scripts/gen_jsonschemas.py).
+
+
+## Related projects:
+
+- <https://github.com/coveooss/json-schema-for-humans>
+- <https://github.com/elisiariocouto/jsonschema-markdown>
+- <https://github.com/adobe/jsonschema2md>
```

Comment:
Can you provide some titles?

Reply:
Done in [1f7de28](https://github.com/sbrunner/jsonschema2md/pull/447/commits/1f7de2840316f0a46655f746cebbe248675576ed).